assets/airports/lipz.json

```diff
@@ -0,0 +1,2329 @@
+            ["45.49771808078429",  "12.337045669555664", "45.497725601029146", "12.337131500244139"],
+            ["45.497725601029146", "12.337131500244139", "45.49776320223832",  "12.337195873260498"],
+            ["45.49776320223832",  "12.337195873260498", "45.49815801341887",  "12.337673306465149"],
+            ["45.502857933880314", "12.343418598175049", "45.50609125134568",  "12.347688674926758"],
+            ["45.50601606002538",  "12.343976497650146", "45.50405353104642",  "12.346991300582886"],
+        
+            
+            ["45.37957255824036", "11.62791313245612", "45.38834000871404", "11.62525116139381"],
+            ["45.39009347869480", "11.62471873540377", "45.39886072804014", "11.62205644659541"],
+            ["45.40061415779897", "11.62152395706588", "45.40938120605340", "11.61886135060853"],
+            ["45.41113459559781", "11.61832879755890", "45.41990144279879", "11.61566587354963"],
+            ["45.42165479213625", "11.61513325699932", "45.43042143832126", "11.61247001553516"],
+            ["45.43217474745928", "11.61193733550359", "45.44094119266585", "11.60927377668162"],
+            ["45.44269446161192", "11.60874103318819", "45.45146070587757", "11.60607715710546"],
+            ["45.45321393463921", "11.60554435016958", "45.46197997800149", "11.60288015692318"],
+            ["45.46373316658622", "11.60234728656426", "45.47249900908272", "11.59968277625124"],
+            ["45.47425215749806", "11.59914984248872", "45.48301779916637", "11.59648501520615"],
+            ["45.48477090741983", "11.59595201805942", "45.49353634829759", "11.59328687390439"],
+            ["45.49528941639670", "11.59275381339288", "45.50405465652156", "11.59008835246246"],
+            ["45.50580768447386", "11.58955522860557", "45.51457272388350", "11.58688945099685"],
+            ["45.51632571169652", "11.58635626381402", "45.52509055042863", "11.58369016962408"],
+            ["45.52684349810992", "11.58315691913471", "45.53560813620224", "11.58049050846063"],
+            ["45.53736104375935", "11.57995719468415", "45.54612548124961", "11.57729046762303"],
+            ["45.54787834869008", "11.57675709057885", "45.55664258561607", "11.57409004722780"],
+            ["45.55839541294746", "11.57355660693533", "45.56715944934696", "11.57088924739144"],
+            ["45.56891223657684", "11.57035574387011", "45.57767607248768", "11.56768806823047"],
+            ["45.57942881962360", "11.56715450149971", "45.58819245508362", "11.56448650986143"],
+            ["45.58994516213316", "11.56395287994065", "45.59870859718022", "11.56128457240084"],
+            ["45.60046126415097", "11.56075087930947", "45.60922449882297", "11.55808225596523"],
+            ["45.61097712572249", "11.55754849972269", "45.61974016005734", "11.55487956067114"],
+            ["45.62149274689323", "11.55434574129685", "45.62829189429773", "11.55227467424585"],
+            ["45.62829189429773", "11.55
```

Comment:
Make sure these brackets are indented correctly and add a new line at the end of the file please